projects/packages/jitm/src/js/jetpack-jitm.js

```diff
@@ -72,13 +72,26 @@ jQuery( document ).ready( function ( $ ) {
 				html +=
 					'<a href="#" data-module="' +
 					envelope.activate_module +
+					'" data-settings_link="' + envelope.settings_link +
 					'" type="button" class="jitm-button is-compact is-primary jptracks" data-jptracks-name="nudge_click" data-jptracks-prop="jitm-' +
 					envelope.id +
 					'-activate_module">' +
 					window.jitm_config.activate_module_text +
 					'</a>';
 				html += '</div>';
 			}
+			if ( envelope.settings_link){
```

Comment:
If this is (in theory) always going to be used as part of the module activation routine, would it make sense to nest this logic in the check for `if ( envelope.active_module )` ?